README.md

```diff
@@ -9,17 +9,47 @@ and HTML tags.
 Autocompletes `[]`, `()`, and `{}`, `""`, `''`, `“”`, `‘’`, `«»`, `‹›`, and
 backticks.
 
-You can toggle whether English/French style quotation marks (`“”`, `‘’`, `«»`
-and `‹›`) are autocompleted via the *Autocomplete Smart Quotes*  setting in the
-settings view.
-
 Use <kbd>ctrl-m</kbd> to jump to the bracket matching the one adjacent to the cursor.
 It jumps to the nearest enclosing bracket when there's no adjacent bracket,
 
 Use <kbd>ctrl-cmd-m</kbd> to select all the text inside the current brackets.
 
 Use <kbd>alt-cmd-.</kbd> to close the current XML/HTML tag.
 
+---
+### Configuration
+
 Matching brackets and quotes are sensibly inserted for you. If you dislike this
 functionality, you can disable it from the Bracket Matcher section of the
 Settings view (<kbd>cmd-,</kbd>).
+
+#### Custom Pairs
+
+You can customize matching pairs in Bracket Matcher at any time. You can do so either Globally via the Settings view (<kbd>cmd-,</kbd>) or at the Scope level via `config.cson` (<kbd>cmd-shift-p</kbd> + "config"). Changes take effect immediately.
+
+* <b>Autocomplete Characters</b> - Comma-separated pairs that the editor will treat as brackets / quotes. Entries in this field override the package defaults.
```

Comment:
Actually, yes: https://github.com/potto007/bracket-matcher/blob/custom-pairs/package.json#L19-L22 

Oops on the bolds. I'll fix that.